components/Header.js

```diff
@@ -3,59 +3,75 @@ import Image from "next/image";
 
 export default function Header() {
   return (
-    <>
+    <StyledHeader>
       <StyledImageWrapper>
         <StyledImage
           src="/countries/jordan.jpg"
-          height={600}
-          width={800}
+          height={450}
+          width={600}
           layout="responsive"
           alt="jordan"
         />
+        <StyledLogo>Travel</StyledLogo>
         <StyledLocation>Jordan</StyledLocation>
         <StyledDate>29/08/23 - 16/09/23</StyledDate>
       </StyledImageWrapper>
-    </>
+    </StyledHeader>
   );
 }
 
-const StyledImageWrapper = styled.div`
-  position: absolute;
+const StyledHeader = styled.header`
+  position: fixed;
   top: 0;
-  left: 50%;
-  transform: translate(-50%);
+  left: 0;
   height: 10rem;
-  width: 360px;
+  width: 100%;
+  @media (min-width: 500px) {
```

Comment:
good idea to make it more accessable but do you need a mediaquery for a mobile app here ? maybe you could change the min-width in the body, so you don't need this part every time :).

Reply:
I don't really need it for the mobile view. I just added the media queries, because I'm mostly working with 100% or 100vw and if someone opens it in web view while doing a code review the design is all over the place. I don't know if there is a way to force the browser to show mobile view.